testsuite/MDAnalysisTests/analysis/test_rms.py

```diff
@@ -165,21 +165,21 @@ def outfile(self, tmpdir):
 
     @pytest.fixture()
     def correct_values(self):
-        return [[0, 0, 0], [49, 49, 4.68953]]
+        return [[0, 1000, 0], [49, 1049, 4.68953]]
 
     @pytest.fixture()
     def correct_values_mass(self):
-        return [[0, 0, 0], [49, 49, 4.74920]]
+        return [[0, 1000, 0], [49, 1049, 4.74920]]
 
     @pytest.fixture()
     def correct_values_group(self):
-        return [[0, 0, 0, 0, 0],
-                 [49, 49, 4.7857, 4.7048, 4.6924]]
+        return [[0, 1000, 0, 0, 0],
+                 [49, 1049, 4.7857, 4.7048, 4.6924]]
 
     @pytest.fixture()
     def correct_values_backbone_group(self):
-        return [[0, 0, 0, 0, 0],
-                [49,   49,   4.6997, 1.9154, 2.7139]]
+        return [[0, 1000, 0, 0, 0],
+                [49,   1049,   4.6997, 1.9154, 2.7139]]
```

Comment:
Can you clarify why results in the `test_rms` suite are changing because of subtle changes in DCD handling machinery?

Reply:
i.e., if `DCD` test file is being used to seed the tests, that might make sense

Reply:
This is the changed time. The DCD file doesn't start at time point 0 but rather at time 1000. We have been treating this wrong for some time.